### PR TITLE
adds transform runtime for polyfills and reintroduces webpack script for dist packaging

### PR DESCRIPTION
**Pull request by mvanlonden**
https://github.com/sstur/react-rte/issues/203

### CODE REVIEW

**Comment by mvanlonden** [comment]
for whatever reason this still doesn't transpile the demo to es5 😕

**Comment by mvanlonden** [comment]
Also I don't see the script or config for generating `dist/react-rte.js`

**Comment by mvanlonden** [comment]
Ok had to root around in the commit history of the webpack config to find how to build dist but it should be good now 👍

File: webpack.config.js [hunk 1]
```diff
@@ -19,12 +20,36 @@ var loaders = [
...
+  },
+  externals: {
+    react: 'react',
+    'react-dom': 'react-dom',
   },
+  module: {loaders: loaders},
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
```
Comment (sstur) [inline webpack.config.js]: This is great. Thank you.

**Comment by bherila** [comment]
Thank you @mvanlonden , sorry it took a while to get this merged!

**Pull request merged**